packages/mongo-nav/src/mongo-select/OrgTrigger.tsx

```diff
@@ -194,10 +121,15 @@ export function OrganizationTrigger({
           aria-disabled={disabled || loading}
           data-testid="org-trigger"
           disabled={disabled || loading}
-          className={cx(baseButtonStyles, orgButtonStyles, {
-            [activeButtonColor]: open,
-            [textLoadingStyle]: loading,
-          })}
+          className={cx(
+            baseButtonStyles,
+            orgButtonStyles,
+            {
+              [activeButtonColor]: open,
+              [textLoadingStyle]: loading,
+            },
+            className,
```

Comment:
(Re above, this would be mitigated if we collapse into one file, but still not at all necessary)